src/test/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterTest.java

```diff
@@ -22,7 +22,7 @@
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-import org.junit.jupiter.api.Test;
+import org.junit.Test;
```

Comment:
Remove this change

Reply:
this change is for using Ignore, which is not support for jupiter.api.Test;